tests/testthat/test-build-home-authors.R

```diff
@@ -37,15 +37,9 @@ test_that("Data authors can accept different filtering", {
   expect_length(data_authors(pkg, roles = "cre")$main, 1)
 })
 
-test_that("Text can be added", {
-  pkg <- as_pkgdown(test_path("assets/sidebar-comment"))
-  expect_null(data_authors_page(pkg)$after)
-  expect_null(data_authors_page(pkg)$before)
-
-  pkg$meta$authors$before <- "Dream team:"
-  pkg$meta$authors$after <- "You are welcome!"
-  expect_equal(data_authors_page(pkg)$before, "<p>Dream team:</p>")
-  expect_equal(data_authors_page(pkg)$after, "<p>You are welcome!</p>")
+test_that("authors data includes inst/AUTHORS", {
+  pkg <- as_pkgdown(test_path("assets/inst-authors"))
```

Comment:
would the test be easier to read if the test package were created here, with some test helper, and if the inst/authors file was then added inline in this test?

I thought there was an issue about test assets but I can't find it.

Reply:
Maybe as part of https://github.com/r-lib/pkgdown/issues/2509

Reply:
Yeah, I think that would probably be the case for many of the tests, but requires quite a bit of test infrastructure investment. We'll see how I go.